### ISSUE TITLE
List of All channels in a Project

### ISSUE THREAD

**Issue by QaiserAbbas** [AgoraIO/Basic-Video-Call#246]
How can we retrieve channel list?

**Comment by yoreland** [comment]
Can you check
https://docs.agora.io/en/rtc/restfulapi/#/Online%20channel%20statistics%20query/channelList

**Comment by QaiserAbbas** [comment]
Yes I have seen this and after using this link I only get an error, that can not fetch

**Comment by QaiserAbbas** [comment]
I have authenticated as described in this link

**Comment by plutoless** [comment]
@QaiserAbbas 
the api shown above is only for admin purposes. it's not intended for your own channel mgmt. you should do it yourself by managing a websocket long connection etc.
